crates/typst/src/eval/func.rs

```diff
@@ -545,8 +545,8 @@ impl<'a> CapturesVisitor<'a> {
             Some(ast::Expr::Import(expr)) => {
                 self.visit(expr.source().to_untyped());
                 if let Some(ast::Imports::Items(items)) = expr.imports() {
-                    for item in items.idents() {
-                        self.bind(item);
+                    for item in items.items() {
```

Comment:
I generally dislike if there's a variable and we access a field or call a method with its exact name. It feels like something wrong is going on. Maybe it's ok here, but maybe we should rename `items` to `iter` since the container itself is already `items`.

Reply:
`iter` definitely sounds nice.